### ISSUE TITLE
Safe Mode should deactivate ALR

### ISSUE THREAD

**Issue by WordPresseur** [wp-media/wp-rocket#7076]
**Describe the bug**
ALR can break sites, but there's no way to deactivate it in the UI. Safe Mode should at least deactivate this feature in order for users to get a functional site meanwhile waiting for support.

**Expected behavior**
Safe Mode in WP Rocket should deactivate ALR too.

**Comment by remyperona** [comment]
Since there is no option for it, it's not that easy. How do we enable it again?

**Comment by jeawhanlee** [comment]
@remyperona Yes I was about to put this in the dev channel. We might need a UI for this to turn off safe mode and an extra option in the DB because up until now, we don't preserve the state of safe mode after being enabled.

CC: @piotrbak

**Comment by WordPresseur** [comment]
<img width="1476" height="703" alt="Image" src="https://github.com/user-attachments/assets/51ad95e7-1e8f-4c23-9989-e744d3800d6d" />
